packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -189,60 +191,95 @@ function getListboxOptionsFromL1Network(network: Chain) {
 
 function NetworkContainer({
   network,
+  balanceFor,
   children
 }: {
   network: Chain
+  balanceFor?: string
   children: React.ReactNode
 }) {
-  const { backgroundImage, backgroundClassName } = useMemo(() => {
-    const { isArbitrum, isArbitrumNova } = isNetwork(network.id)
+  const { backgroundImage, backgroundClassName, customAddressBannerClassName } =
```

Comment:
NIT: maybe not in this PR, but this file is getting quite long and hard to read. we might want to extract some components